application/models/GoogleSheets_Model.php

```diff
@@ -88,16 +88,18 @@ public function highlightRow($row_num, $colour)
      *
      * @param string $email The email of the user to record.
      * @param string $fullName The full name of the user.
+     * @param string @userUpi The UPI of the user. 
+     * @param string @userUid the ID number of the user. 
      * @param string $paymentMethod The type of payment the user is selecting.
      */
-    public function addNewRecord($email, $fullName, $paymentMethod)
+    public function addNewRecord($email, $fullName, $userUpi, $userUid, $paymentMethod)
```

Comment:
Remove `userUid` as above.

Also, let's rename `userUpi` to just be `upi`, just to be consistent with the naming conventions in the rest of the places. We also don't refer to `userFullName` or `userEmail`.